src/gpb_names.erl

```diff
@@ -702,6 +723,12 @@ rename_fields(RF, Fields, Defs) ->
       end,
       Fields).
 
+rename_enumerators(RF, Enumerators) ->
```

Comment:
rename_enum_fields might be better name?

Reply:
done.